### PR TITLE
shipping.hcov19_observation_v1: Add swab sample type and media type

### PR DESCRIPTION
**Pull request by kschwabefry**
Add sample type and media type (matrix collection) columns to
shipping view, as requested by IDM

### CODE REVIEW

**Comment by kschwabefry** [comment]
[Add swab type, media type to hcov19_observation_v1 view](https://trello.com/c/xU9B1zKN/858-add-swab-type-media-type-to-hcov19observationv1-view)

**Review by kairstenfay — APPROVED** [review]
Looks good to me -- don't forget to tag the changes with `sqitch tag`.

File: schema/deploy/shipping/views.sql [hunk 1]
```diff
@@ -1080,7 +1080,9 @@ create or replace view shipping.hcov19_observation_v1 as
         sex,
 
         -- Misc cruft
-        sample.details->>'sample_origin' as manifest_origin
+        sample.details->>'sample_origin' as manifest_origin,
+        sample.details->>'swab_type' as swab_type,
+        sample.details->>'collection_matrix' as collection_matrix
```
Comment (kairstenfay) [inline schema/deploy/shipping/views.sql:1085]: Hah I just realized these are under Tom's comment calling this "Misc cruft". I don't think swab type and collection matrix fall under "cruft". Might be worth moving those details above the comment or removing the comment altogether.

**Review by joverlee521 — APPROVED** [review]

**Review by jccraft-uw — APPROVED** [review]

File: schema/deploy/shipping/views.sql (hunk 1 above)
Comment (kschwabefry) [reply schema/deploy/shipping/views.sql:1085]: Pushing new commit to update to `--sample details columns`. I'm not sure it's that informative, but along wIth the other comments in this view, it at least allows for logical grouping of variables and easier scanning.

**Comment by kschwabefry** [comment]
Deployed

**Pull request merged**